ngraph/frontend/frontend_manager/CMakeLists.txt

```diff
@@ -14,11 +14,24 @@ source_group("src" FILES ${LIBRARY_SRC})
 source_group("include" FILES ${LIBRARY_HEADERS})
 source_group("public include" FILES ${LIBRARY_PUBLIC_HEADERS})
 
-# Create shared library
+# Static library
+
+add_library(${TARGET_NAME}_static STATIC ${LIBRARY_SRC} ${LIBRARY_HEADERS} ${LIBRARY_PUBLIC_HEADERS})
+add_library(ngraph::${TARGET_NAME}::static ALIAS ${TARGET_NAME}_static)
```

Comment:
can we have `ngraph::${TARGET_NAME}::static` as a base name for `add_library(.. STATIC)`? will it work without `ALIAS`?

Reply:
CMake shows error for such name that "it is reserved or not valid for certain CMake features....", I also see target names with "::" separator used for aliases only